ArduCopter/sensors.cpp

```diff
@@ -461,3 +471,34 @@ void Copter::update_beacon()
 {
     g2.beacon.update();
 }
+
+// init visual odometry sensor
+void Copter::init_visual_odom()
+{
+#if VISUAL_ODOMETRY_ENABLED == ENABLED
+    g2.visual_odom.init();
+#endif
+}
+
+// update visual odometry sensor
+void Copter::update_visual_odom()
+{
+#if VISUAL_ODOMETRY_ENABLED == ENABLED
+    // check for updates
+    if (g2.visual_odom.enabled() && (g2.visual_odom.get_last_update_ms() != visual_odom_last_update_ms)) {
+        visual_odom_last_update_ms = g2.visual_odom.get_last_update_ms();
+        float time_delta_sec = g2.visual_odom.get_time_delta_usec() / 1000000.0f;
```

Comment:
const time_delta_sec.
Better to use * 0.000001f for performance reason normally but it could end to use a big float with a really small float ... I will ask what the better